docs/data/rpc/api-reference/methods/sendTransaction.mdx

```diff
@@ -4,5 +4,143 @@ description: Submits a transaction
 ---
 
 import { RpcMethod } from "@site/src/components/RpcMethod";
+import { CodeExample } from "@site/src/components/CodeExample";
 
 <RpcMethod method="sendTransaction" platform="soroban" />
+
+### SDK Guide
+
+The example above is sending a transaction using RPC methods directly. If you are using the Stellar SDK to build applications, you can use the native functions to get the same information.
+
+<div>
```

Comment:
Let's remove this `<div>` (and the corresponding, `</div>`, of course). It shouldn't be necessary for this page.

Reply:
I added it because the code sample was flexing to the right

Reply:
oh, you're right!! That's something (entirely separate from your PR) I have been meaning to address. Thanks for pointing it out!!